content/en/docs/tutorials/configuration/kubelet-standalone.md

```diff
@@ -0,0 +1,474 @@
+
+## {{% heading "objectives" %}}
+
+* Install and configure `containerd`, `cni-plugins`, and `kubelet` on a Linux system.
+* Run `containerd` and `kubelet` as `systemd` services.
+* Launch a Pod running `nginx` that listens to requests on TCP port 80 on the Pod's IP address.
+
+## {{% heading "prerequisites" %}}
+
+- Admin (root) access to a Linux system (virtual, cloud, or physical instance) running `systemd`.
+- Access to the Internet to download the various components needed for the tutorial, such as:
+    - A {{< glossary_tooltip text="container runtime" term_id="container-runtime" >}}
+that implements the Kubernetes {{< glossary_tooltip term_id="cri" text="(CRI)">}}.
+    - Network plugins {{< glossary_tooltip text="Container Networking Interface (CNI)" term_id="cni" >}}
+- Required CLI tools: `curl`, `tar`.
+- Suggested CLI tools: `jq`.
+
+<!-- lessoncontent -->
+
+## Download, install, and configure the components
+
+{{% thirdparty-content %}}
+
+{{< note >}}
+Download the latest available versions of the required packages.
+The relative path to the downloaded software is assumed to be the `sources` directory.
+{{< /note >}}
+
+{{< note >}}
+Run all the commands as the `root` user.
+{{< /note >}}
+
+
+```shell
+mkdir sources
+cd sources/
+```
+
+### Network Plugins
+
+This tutorial suggests downloading CNI network plugins.
+Visit [releases](https://github.com/containernetworking/plugins/releases) (external link).
+
+{{< tabs name="download_cni_plugins" >}}
+{{< tab name="x86-64" codelang="bash" >}}
+curl -LO https://github.com/containernetworking/plugins/releases/download/v1.5.1/cni-plugins-linux-amd64-v1.5.1.tgz
+{{< /tab >}}
+{{< tab name="ARM64" codelang="bash" >}}
+curl -LO https://github.com/containernetworking/plugins/releases/download/v1.5.1/cni-plugins-linux-arm64-v1.5.1.tgz
+{{< /tab >}}
+{{< /tabs >}}
+
+<!--
+```shell
+curl -LO https://github.com/containernetworking/plugins/releases/download/v1.5.1/cni-plugins-linux-amd64-v1.5.1.tgz
+```
+-->
+
+Install:
+
+```shell
+mkdir -p /opt/cni/bin
+```
+
+{{< tabs name="install_cni_plugins" >}}
+{{< tab name="x86-64" codelang="bash" >}}
+tar Cxvf /opt/cni/bin/ cni-plugins-linux-amd64-v1.5.1.tgz
+{{< /tab >}}
+{{< tab name="ARM64" codelang="bash" >}}
+tar Cxvf /opt/cni/bin/ cni-plugins-linux-arm64-v1.5.1.tgz
+{{< /tab >}}
+{{< /tabs >}}
+
+<!--
+```shell
+mkdir -p /opt/cni/bin
+tar Cxvf /opt/cni/bin/ cni-plugins-linux-amd64-v1.5.1.tgz
+```
+-->
+
+Quick test:
+
+```shell
+/opt/cni/bin/host-local --version
+```
+
+The output is similar to:
+
+```
+CNI host-local plugin v1.5.1
+CNI protocol versions supported: 0.1.0, 0.2.0, 0.3.0, 0.3.1, 0.4.0, 1.0.0
+```
+
+Configure:
+
+```shell
+mkdir -p /etc/cni
```

Comment:
This is likely to need `sudo` to actually work.

Reply:
Done